src/plugin/plugin.cpp

```diff
@@ -83,6 +84,7 @@ class Q_DECL_EXPORT NemoEmailPlugin : public QQmlExtensionPlugin
         qmlRegisterType<AttachmentListModel>(uri, 0, 1, "AttachmentListModel");
         qmlRegisterUncreatableType<FolderAccessor>(uri, 0, 1, "FolderAccessor",
                                                    "FolderAccessor is created via FolderListModel or similar");
+        qmlRegisterType<EmailContactModel>(uri, 0, 1, "EmailContactModel");
```

Comment:
Since this is not actually a generic email contact model but rather a model for recent email contacts, the name could indicate that. RecentEmailContactModel or something like that?